lib/cartopy/mpl/ticker.py

```diff
@@ -422,6 +446,10 @@ def _format_degrees(self, deg):
         return _PlateCarreeFormatter._format_degrees(self, self._fix_lons(deg))
 
     def _hemisphere(self, value, value_source_crs):
+
+        if not self._direction_labels:
```

Comment:
You shouldn't need these anymore since it is taken care of in the PlateCarree class?